libs/FScreen.c

```diff
@@ -409,9 +409,13 @@ scan_screens(Display *dpy)
 	for (i = 0; i < n; i++) {
 
 		char *name = XGetAtomName(dpy, rrm[i].name);
+		Bool must_free_name = True;
```

Comment:
As an aside here, the "Bool" type is usually reserved for Xlib operations, and either "int" or "bool" (via `stdbool.h`) can be used instead.